src/Aspire.Hosting.Dapr/DaprDistributedApplicationLifecycleHook.cs

```diff
@@ -105,10 +104,6 @@ public async Task BeforeStartAsync(DistributedApplicationModel appModel, Cancell
                         fileName,
                         Command("run"),
                         daprAppPortArg(sidecarOptions?.AppPort),
-                        daprGrpcPortArg(sidecarOptions?.DaprGrpcPort?.ToString(CultureInfo.InvariantCulture)),
```

Comment:
Was there a reason the removed code was specifying invariant culture (it looks like that was added in your #1184, though I don't immediately see why it'd be necessary)? I only ask because the string interpolation version that you note already handles it doesn't specify invariant culture.

Reply:
The other code relies on DCP to output port in a culture-insensitive way.

It might have been an overkill in this case, but I have been burned too many times by I18n issues that I am now conditioned to specify the culture explicitly whenever a conversion to string occurs.